### PR TITLE
feat(docs): add python sdk connect to deployments docs

### PR DESCRIPTION
**Pull request by wuzufeng**

### CODE REVIEW

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 1]
```diff
@@ -1,37 +1,237 @@
...
+本文主要介绍如何在 Python 项目中使用 **paho-mqtt** 客户端库 ，实现客户端与 MQTT 服务器的连接、订阅、取消订阅、收发消息等功能。
 
+[Python](https://www.python.org/) 是一种广泛使用的解释型、高级编程、通用型编程语言。
+Python 的设计哲学强调代码的可读性和简洁的语法（尤其是使用空格缩进划分代码块，而非使用大括号或者关键词）。
+Python 让开发者能够用更少的代码表达想法，不管是小型还是大型程序，该语言都试图让程序的结构清晰明了。
 
+## 前提条件
 
-## 先决条件
+已经创建了部署，在部署概览下可以查看到连接相关的信息，同时你可以了使用WebSocket先测试连接到MQTT服务器。
```
Comment (Adek06) [inline zh_CN/connect_to_deployments/python_sdk.md:11]: 中英文间空隔

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 2]
```diff
@@ -1,37 +1,237 @@
...
+本项目使用 Python 3.8 进行开发测试，可用如下命令确认 Python 的版本。
 
-* Python 3.4+ 版本
+```
+➜  ~ python3 --version             
+Python 3.8.6
+```
 
-* 已安装 **paho-mqtt** MQTT 客户端。如未安装，请运行如下命令完成安装
+## 安装使用客户端
```
Comment (Adek06) [inline zh_CN/connect_to_deployments/python_sdk.md:20]: `安装使用客户端` 改成 `安装 MQTT 客户端` 好点吧

Comment (wuzufeng) [reply zh_CN/connect_to_deployments/python_sdk.md:20]: @azuregaya 这个是李研模板上的，需要修改吗

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 3]
```diff
@@ -1,37 +1,237 @@
...
+
+![python_pub](./_assets/python_pub.png)
+
+### 消息订阅
+
+![python_sub](./_assets/python_sub.png)
+
+## 下一步
+
+以上为您演示了如何使用paho.mqtt.m2mqtt 客户端库连接到EMQ X Cloud，可以在 [这里](https://github.com/emqx/MQTT-Client-Examples/tree/master/mqtt-client-Python3/) 下载到示例的源码。
```
Comment (Adek06) [inline zh_CN/connect_to_deployments/python_sdk.md:236]: 中英文间空隔

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 4]
```diff
@@ -1,37 +1,237 @@
...
+![python_pub](./_assets/python_pub.png)
+
+### 消息订阅
+
+![python_sub](./_assets/python_sub.png)
+
+## 下一步
+
+以上为您演示了如何使用 paho.mqtt.m2mqtt 客户端库连接到 EMQ X Cloud，可以在 [这里](https://github.com/emqx/MQTT-Client-Examples/tree/master/mqtt-client-Python3/) 下载到示例的源码。
+同时也可以在 [GitHub](https://github.com/emqx/MQTT-Client-Examples) 上找到更多起他语言的 Demo 示例。
```
Comment (Adek06) [inline zh_CN/connect_to_deployments/python_sdk.md:237]: `起他` -> `其它`

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 5]
```diff
@@ -1,37 +1,237 @@
...
+本文主要介绍如何在 Python 项目中使用 **paho-mqtt** 客户端库 ，实现客户端与 MQTT 服务器的连接、订阅、取消订阅、收发消息等功能。
 
+[Python](https://www.python.org/) 是一种广泛使用的解释型、高级编程、通用型编程语言。
+Python 的设计哲学强调代码的可读性和简洁的语法（尤其是使用空格缩进划分代码块，而非使用大括号或者关键词）。
+Python 让开发者能够用更少的代码表达想法，不管是小型还是大型程序，该语言都试图让程序的结构清晰明了。
 
+## 前提条件
 
-## 先决条件
+已经创建了部署，在部署概览下可以查看到连接相关的信息，同时你可以了使用 WebSocket 先测试连接到 MQTT 服务器。
```
Comment (Adek06) [inline zh_CN/connect_to_deployments/python_sdk.md:11]: `同时你可以了使用`

多了一个了

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 6]
```diff
@@ -1,37 +1,236 @@
...
+
+![python_pub](./_assets/python_pub.png)
+
+### 消息订阅
+
+![python_sub](./_assets/python_sub.png)
+
+## 下一步
+
+以上为您演示了如何使用 paho.mqtt.m2mqtt 客户端库连接到 EMQ X Cloud，可以在 [这里](https://github.com/emqx/MQTT-Client-Examples/tree/master/mqtt-client-Python3/) 下载到示例的源码。
```
Comment (Zhiw) [inline zh_CN/connect_to_deployments/python_sdk.md:235]: 这里没改 paho.mqtt.m2mqtt

File: zh_CN/connect_to_deployments/python_sdk.md [hunk 7]
```diff
@@ -1,37 +1,236 @@
...
+    def on_connect(client, userdata, flags, rc):
+        if rc == 0:
+            print("Connected to MQTT Broker!")
+        else:
+            print("Failed to connect, return code %d\n", rc)
+
+    client = mqtt_client.Client(client_id)
+    client.username_pw_set(username, password)
+    client.on_connect = on_connect
+    client.connect(broker, port)
```
Comment (zibuyu1995) [inline zh_CN/connect_to_deployments/python_sdk.md:141]: 看一下 clean session 默认是否为 Ture，不为 True 需要设置为 True

**Review by zibuyu1995 — CHANGES_REQUESTED** [review]

**Review by zibuyu1995 — APPROVED** [review]

File: zh_CN/connect_to_deployments/python_sdk.md (hunk 2 above)
Comment (azuregaya) [reply zh_CN/connect_to_deployments/python_sdk.md:20]: 可以改成 `安装 MQTT 客户端`

Comment (wuzufeng) [reply zh_CN/connect_to_deployments/python_sdk.md]: @Adek06 @azuregaya 已修改

File: en_US/connect_to_deployments/python_sdk.md [hunk 8]
```diff
@@ -1,37 +1,241 @@
...
 
 
-## Precondition
+## Prerequisites
 
-* Already [Create Deployment](../deployments/create_deployment.md) in EMQ X Cloud, and the deployment status is **running**
+The deployment has been created, you can see connection information under the Deployment Overview, and you can test connecting to the MQTT server using WebSocket.
 
-* Python 3.4+ version
+This project uses Python 3.8 to develop and test. Readers can use the following command to confirm the Python version.
```
Comment (azuregaya) [inline en_US/connect_to_deployments/python_sdk.md:14]: This project uses Python 3.8 to develop and test. Confirm the Python version by the following command.

File: en_US/connect_to_deployments/python_sdk.md [hunk 9]
```diff
@@ -1,37 +1,241 @@
...
-* [Connect to deployments and publish and subscribe to messages with Websocket over TLS/SSL protocol](https://github.com/emqx/MQTT-Client-Examples/blob/master/mqtt-client-Python3/pub_sub_wss.py)
+2. Import the Paho MQTT client
+
+```python
+from paho.mqtt import client as mqtt_client
+```
+
+## Connect to the MQTT server
+
+Please find the address and port information in the Deployment Overview.
```
Comment (azuregaya) [inline en_US/connect_to_deployments/python_sdk.md:42]: address -> host

File: en_US/connect_to_deployments/python_sdk.md [hunk 10]
```diff
@@ -1,37 +1,241 @@
...
+
+### Connection Settings
+
+Set the address, port and topic of MQTT Broker connection. At the same time, we call the Python function 'random.randint' to randomly generate the MQTT client id.
```
Comment (azuregaya) [inline en_US/connect_to_deployments/python_sdk.md:46]: address -> host

File: en_US/connect_to_deployments/python_sdk.md [hunk 11]
```diff
@@ -1,37 +1,241 @@
...
+
+if __name__ == '__main__':
+    run()
+```
+
+## Test
+
+### Publish messages
+
+Run the code of publishing messages, we will see that the client connects successfully and publishes messages successfully
```
Comment (azuregaya) [inline en_US/connect_to_deployments/python_sdk.md:229]: Run the code of publishing messages, we will see that the client connects and publishes messages successfully.

File: en_US/connect_to_deployments/python_sdk.md [hunk 12]
```diff
@@ -1,37 +1,241 @@
...
+
+![python_pub](./_assets/python_pub.png)
+
+### Subscribe
+
+![python_sub](./_assets/python_sub.png)
+
+## Next step
+
+This shows you how to use the paho-mqtt client library to connect to the EMQ X Cloud.
```
Comment (azuregaya) [inline en_US/connect_to_deployments/python_sdk.md:239]: All of the above shows that how to use the paho-mqtt client library to connect to the EMQ X Cloud.

Comment (wuzufeng) [reply en_US/connect_to_deployments/python_sdk.md]: @azuregaya 已修改

**Review by Zhiw — APPROVED** [review]

**Pull request merged**